force-app/main/default/layouts/Requirement__c-Requirement Layout.layout-meta.xml

```diff
@@ -139,14 +139,14 @@
             <sortOrder>4</sortOrder>
         </platformActionListItems>
         <platformActionListItems>
-            <actionName>FeedItem.TextPost</actionName>
+            <actionName>SendEmail</actionName>
```

Comment:
This change looks like the buttons got reordered, but it `FeedItem.TextPost` and `SendEmail` are actually still in the same order as before. The metadata just had `<sortOrder>6</sortOrder>` listed before `<sortOrder>5</sortOrder>` previously